enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+ 
+ **NOTE** To know how it can be implemented check [here](https://github.com/jmccormick2001/indexdump/blob/main/gethttp.go) a code example of a similar implementation.
+
+**Gathering the info from PROJECT file**
+
+We can check if the Builder is `Kubebuilder` or `SDK` and its other aspects by looking into the `PROJECT` file see:
+
+- If the attribute `layout` of the `PROJECT` file be `ansible.sdk.operatorframework.io/vX` (see e.g [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/ansible/memcached-operator/PROJECT#L2)):
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK` |
+| Language | `ansible` |
+| Project Layout | `ansible.sdk.operatorframework.io/vX` |
+| SDK Version | Check in the Dockerfile. (e.g see [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/ansible/memcached-operator/Dockerfile#L1) where it will be `v1.4.2`). |
+
+- If the attribute `layout` of the `PROJECT file be `helm.sdk.operatorframework.io/vX` (see e.g [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/helm/memcached-operator/PROJECT#L2)):
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK` |
+| Language | `helm` |
+| Project Layout | `helm.sdk.operatorframework.io/vX` |
+| SDK Version | Check in the Dockerfile. (e.g see [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/helm/memcached-operator/Dockerfile#L1) where it will be `v1.4.2`) |
+
+- If the attribute `layout of the`PROJECT` file be `go.kubebuilder.io/vX` (see e.g [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/go/v3/memcached-operator/PROJECT#L2)):
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | Check if it has plugins with `sdk.operatorframework.io/` (e.g see [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/go/v3/memcached-operator/PROJECT#L12-L14)) to set as `SDK` otherwise it will be `Kubebuilder`|
+| Language | `golang` |
+| Project Layout | `go.kubebuilder.io/vX` |
+
+However, note that for the previous versions of projects built with `Kubebuilder` we will not found the layout information in the `PROJECT` file as well. (e.g see [here](https://github.com/kubernetes-sigs/kubebuilder/blob/v2.3.2/testdata/project-v2/PROJECT)) then:
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `Kubebuilder`|
+| Language | `golang` |
+| Project Layout | If the attribute `version` in the `PROJECT` file be `2` it will be `go.kubebuilder.io/v2` otherwise, it will be `go.kubebuilder.io/v1` |
+
+**Gathering the info from go.mod file**
+
+Now, if the repository has not a `PROJECT` file then, it means that the project could also be built with an SDK version < 1.0 or any other tool which is not SDK or Kubebuilder. In this way, check if the project has a `go.mod` file with the sdk dependency `github.com/operator-framework/operator-sdk`: 
```

Comment:
I doubt this is feasible, since there is no standard field within a bundle pointing to a version-controlled repository. This is probably a non-goal.

Reply:
We do that in the POC: https://github.com/jmccormick2001/indexdump/blob/main/gethttp.go#L69
See here in its result we obtaining the 0.17.0 value for example https://github.com/jmccormick2001/indexdump/blob/main/report.txt.11-11-2020#L13

So, yes, it is possible. Look that we indeed linking the POC in the proposal already to help who will implement it check code examples over how to achieve that.

Reply:
Oh I see what this section is saying. So you want the `audit` command to make external network requests? That's definitely something that should be documented in command docs, and have graceful failure for running in an airgapped env.

Reply:
Yes. I think it was detailed covered here. Really thank you for point out. So, I am closing this one. However, please let me know if you still think that it is missing any detail info.